packages/ui-svelte/src/Code.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+  import clsx from 'clsx'
+  export let className = ''
+</script>
+
+<code class={clsx('root', className)}>
```

Comment:
I'm torn about this. I understand _why_ there's a desire to create this sort of wrapper component (it's a very common mindset in the JS-centric React community), but it's fairly unidiomatic — it's not considered a best practice in Svelte. It's much more common to just use `<code>` rather than `<Code>`, if all the component is doing is adding some styles that could live in a `base.css`.

Firstly, it's much more efficient, both in terms of authoring (no need to import the component, just write `<code>`) and app performance (no need to create potentially hundreds of extra components, no need to run `clsx(...)` for each one, no need to include reactivity code to anticipate those classnames changing when they probably won't).

Secondly, it works with content in HTML, such as you might get from a markdown document or a CMS.

Thirdly, because of how style encapsulation is designed to work, these will behave differently:

```svelte
<code class="extra">here is my code</code>

<style>
  .extra {
    color: hotpink;
  }
</style>
```

```svelte
<script>
  import { Code } from '@vercel/examples-ui-svelte';
</script>

<Code className="extra">here is my code</Code>

<style>
  .extra {
    color: hotpink;
  }
</style>
```

The second example won't work. (This is a frequent subject of debate, but it's deliberate and unlikely to change any time soon!)

So to be honest I think it would be better if this component (and `<Link>`, `<Snippet>` and `<Text>`) didn't exist.